digitalocean/resource_digitalocean_database_replica.go

```diff
@@ -0,0 +1,139 @@
+package digitalocean
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/digitalocean/godo"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceDigitalOceanDatabaseReplica() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDigitalOceanDatabaseReplicaCreate,
+		Read:   resourceDigitalOceanDatabaseReplicaRead,
+		Delete: resourceDigitalOceanDatabaseReplicaDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+
+			"cluster_id": {
+				Type:         schema.TypeString,
+				Computed:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+
+			"region": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"size": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceDigitalOceanDatabaseReplicaCreate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*CombinedConfig).godoClient()
+	clusterId := d.Get("cluster_id").(string)
+
+	opts := &godo.DatabaseCreateReplicaRequest{
+		Name:   d.Get("name").(string),
+		Region: d.Get("region").(string),
+		Size:   d.Get("size").(string),
+	}
+
+	log.Printf("[DEBUG] DatabaseReplica create configuration: %#v", opts)
+	replica, _, err := client.Databases.CreateReplica(context.Background(), clusterId, opts)
+	if err != nil {
+		return fmt.Errorf("Error creating DatabaseReplica: %s", err)
+	}
+
+	replica, err = waitForDatabaseReplica(client, clusterId, "online", replica.Name)
+	if err != nil {
+		return fmt.Errorf("Error creating DatabaseReplica: %s", err)
+	}
+
+	log.Printf("[INFO] DatabaseReplica Name: %s", replica.Name)
+
+	return resourceDigitalOceanDatabaseClusterRead(d, meta)
```

Comment:
Missed this one on my first pass.

```suggestion
	return resourceDigitalOceanDatabaseReplicaRead(d, meta)
```